mix/ClientModel.cpp

```diff
@@ -307,6 +308,14 @@ void ClientModel::executeSequence(vector<TransactionSettings> const& _sequence,
 			m_gasCosts.clear();
 			for (TransactionSettings const& transaction: _sequence)
 			{
+				QString contractName = resolveContractName(transaction.contractId);
+				QString address = resolveToken(transaction.contractId, deployedContracts);
+				if (transaction.functionId == "(transfert)")
```

Comment:
Can we make it without a magic string constant? Using empty string or explicit flag